apps/signer/main.c

```diff
@@ -209,6 +214,10 @@ main(gint argc, gchar *argv[])
     goto out;
   }
 
+  // Convert recurrence string to int and set property
+  gint recurrence_int = atoi(recurrence_str);
+  g_object_set(G_OBJECT(signedvideo), "recurrence", recurrence_int, NULL);
```

Comment:
Only set if the user did set it. You are overriding the library default. Even if the defaults match each other now, we would need to keep them in sync for the rest of our lives.
```
if (recurrence > 0) {
  g_object_set(G_OBJECT(signedvideo), "recurrence", recurrence, NULL);
}
```